### PR TITLE
run rsync in daemon mode

### LINKED ISSUE #86: Use rsync/stunnel without ssh in the middle
With the introduction of stunnel we're encrypting encrypted traffic. We should be able to safely remove ssh and start an rsync daemon on the destination pod, however this will involve setting up an rsyncd.conf with modules for each volume I believe. That should look something like

```
uid = root
gid = root

[volume1]
path = /volume1
read only = false

...
```

The instead of running sshd, `/usr/bin/rsync --daemon --no-detach`

### PR DESCRIPTION
**Pull request by alaypatel07**
This PR brings in fixes to:
- convert stunnel certs creation from bash to ansible alaypatel07/pvc-migrate#2
- runs rsync in daemon mode, which fixes #86

### CODE REVIEW

File: 2_pvc_destination_gen/extras/container/stunnel.conf [hunk 1]
```diff
@@ -4,9 +4,8 @@ socket = l:TCP_NODELAY=1
 socket = r:TCP_NODELAY=1
 sslVersion = TLSv1.2
 
-[ssh]
-TIMEOUTclose = 0
```
Comment (pranavgaikwad) [inline 2_pvc_destination_gen/extras/container/stunnel.conf:8]: Any consequences of removing `TIMEOUTclose` ?

File: 3_run_rsync/vars/defaults.yml [hunk 2]
```diff
@@ -10,5 +10,8 @@ successful_pvcs_dir: ../output/
 transfer_pod_image: "quay.io/konveyor/stunnel-ssh-transfer:latest"
 mig_dest_ssh_user: root # note: image above has been designed to work with login as root
 
-stunnel_cert_filepath: "./scripts/tls.crt"
-stunnel_key_filepath: "./scripts/tls.key"
+stunnel_cert_filepath: "./files/tls.crt"
+stunnel_key_filepath: "./files/tls.key"
+
+rsyncd_port:  22
+debug_password: true
```
Comment (pranavgaikwad) [inline 3_run_rsync/vars/defaults.yml:17]: Is `debug_password` required?

**Review by pranavgaikwad — COMMENTED** [review]
Left 2 questions. Otherwise, LGTM

File: 2_pvc_destination_gen/extras/container/stunnel.conf (hunk 1 above)
Comment (alaypatel07) [reply 2_pvc_destination_gen/extras/container/stunnel.conf:8]: I dont know, I thought that was a `[ssh]` module-specific, I was not sure if it is available in `[rsync]`

File: 3_run_rsync/vars/defaults.yml (hunk 2 above)
Comment (alaypatel07) [reply 3_run_rsync/vars/defaults.yml:17]: yes, for debugging our envs now, can default this to false or remove right before merging to master

File: 2_pvc_destination_gen/extras/container/stunnel.conf (hunk 1 above)
Comment (alaypatel07) [reply 2_pvc_destination_gen/extras/container/stunnel.conf:8]: fixed it @pranavgaikwad

**Review by pranavgaikwad — APPROVED** [review]
ACK

**Pull request merged**